CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
-### 4.1.1 (Next)
+### 4.2.0 (Next)
 
 * [#239](https://github.com/mongoid/mongoid-rspec/pull/239): Add support for if/unless validator options - [@knovoselic](https://github.com/knovoselic).
-* [#244](https://github.com/mongoid/mongoid-rspec/pull/123): Migration from TravisCI to GitHub Actions - [@skalibog](https://github.com/skalibog).
+* [#244](https://github.com/mongoid/mongoid-rspec/pull/244): Migration from TravisCI to GitHub Actions - [@skalibog](https://github.com/skalibog).
+* [#245](https://github.com/mongoid/mongoid-rspec/pull/245): Added Support for Mongoid v9.0.0 - [@saisrinivasan](https://github.com/SairamSrinivasan).
```

Comment:
Let's match the rest of CHANGELOG: "Add support for Mongoid 9"  (Add vs. Added, lowercase support, generalize 9).